src/index.html

```diff
@@ -3,10 +3,42 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <title>Calendar</title>
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
+      <div class="day day-1"></div>
+      <div class="day day-2"></div>
+      <div class="day day-3"></div>
+      <div class="day day-4"></div>
```

Comment:
```suggestion
      <div class="day"></div>
      <div class="day"></div>
      <div class="day"></div>
      <div class="day"></div>
```
циклом роздай всім правильний контент